core/document-types/page-base.php

```diff
@@ -69,11 +69,11 @@ public function get_css_wrapper_selector() {
 	protected function register_controls() {
 		parent::register_controls();
 
-		self::register_hide_title_control( $this );
+		static::register_hide_title_control( $this );
 
-		self::register_post_fields_control( $this );
+		static::register_post_fields_control( $this );
 
-		self::register_style_controls( $this );
+		static::register_style_controls( $this );
```

Comment:
Why did we need this change?

Reply:
@louiswol94 this is needed because I need to only show the controls for site title and page status in Page Settings. So the easiest way is to overrid this methods and leave them empty. that's why I need static. If you don't like the approach I can do it differently and simply override the `register_controls()` method